<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LightsTab</class>
 <widget class="QWidget" name="LightsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCheckBox" name="brakeLights">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Brake Lights</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="bmsStrobeLight">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>BMS Strobe Light</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lowBeams">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Low Beams</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="highBeams">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>High Beams</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="rightSignal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Right Signal</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="lightsAlive">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>92</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Alive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="leftSignal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Left Signal</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
